mitodl/mitxonline
# Programs API v2

# What are the relevant tickets?
Closes https://github.com/mitodl/hq/issues/2802

# Description (What does it do?)
- Namespaces the v1 and v2 APIs
- refactors the API code into v2 and v2
- Removes courses from the programs endpoint

# How can this be tested?
Check that tests run.  You'll notice that we're doing far less queries than we were on the current main branch.
Hit the endpoint (http://mitxonline.odl.local:8013/api/v2/programs/) and verify the structure is the same as main branch but with flat list of IDs for courses rather than full serialized objects.
